src/logging.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import * as winstonLogger from './Adapters/Logger/WinstonLogger';
+
+let logger = winstonLogger.logger;
+export function setLogger(aLogger) {
+  if (!aLogger) {
+    console.log('NO LOGGER SET!');
+  }
+  logger = aLogger;
+}
+
+function getLogger() {
+  return logger;
+}
+
+export function addTransport(transport) {
+  winstonLogger.addTransport(transport);
+}
+
+let logging = {
+  setLogger,
+  getLogger,
+  addTransport
+};
+
+Object.defineProperty(logging, 'logger', {
+  get: getLogger
+});
+
+module.exports = logging;
+export default logging;
```

Comment:
I don't know if this is possible but may be more convenient to export logger/getLogger as the default as that is used most often and export logging (or just setLogger) as something else.